types/ember-data/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        "ember-source": "~5.1.0"
```

Comment:
@chriskrycho does this seem right? or should we be doing a peer-dep and ranges? + dev-dep for the tests here?